config/routes.yaml

```
@@ -51,3 +51,8 @@ app_figure_update:
 app_figure_delete:
     path: /trick/delete/{slug}
```

```suggestion
    path: /trick/{slug}
```

The verb delete specify already your action.